charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -75,6 +75,9 @@ spec:
             - "--drivername=$(DRIVER_NAME)"
 {{- if .Values.topology.enabled }}
             - "--domainlabels={{ .Values.topology.domainLabels | join "," }}"
+{{- end }}
+{{- if .Values.nodeplugin.enableprofiling.enabled }}
+            - "--enableprofiling={{ .Values.nodeplugin.enableprofiling.enabled }}"
```

Comment:
```suggestion
- "--enableprofiling={{ .Values.nodeplugin.profiling.enabled }}"
```

Reply:
Please take care of this in other places also for the helm. and also Enable this in helm testing lets see if nothing breaks.

Reply:
sure. Will update it.